libs/extensions/angular/image-banner/src/image-banner.component.stories.ts

```diff
@@ -107,6 +122,57 @@ export const NoDismiss: Story = {
   }),
 };
 
+/**
+ * By default the title is truncated with an ellipsis when it exceeds the width of the banner, and body text can be a maximum of three lines.
+ * If these defaults are unwanted custom `title` and `bodyText` markup can instead be provided to the image banner which can be styled as needed.
+ */
+export const customContent: Story = {
+  args: {
+    actionButtonText: 'Read more',
+    imagePath: 'assets/images/leaves.jpg',
+    dismissClick: undefined,
+  },
+
+  render: (args) => ({
+    props: args,
+    styles: [
+      ' .ellipsis{ white-space: nowrap;overflow: hidden;text-overflow: ellipsis;}',
+      ' .no-ellipsis {white-space:wrap}',
+    ],
+    template: `
+    <kirby-x-image-banner ${argsToTemplate(args)}>
+      <div class="no-ellipsis" title>The title is very long and will NOT be truncated with an ellipsis when it exceeds the width of the banner</div>
+      <div class="ellipsis" bodyText>A lot of text that should be truncated with an ellipsis when it exceeds the width of the banner.</div>
+    </kirby-x-image-banner>
+    `,
+  }),
+};
+
+/**
+ * The components min-height can be overriden with the custom css property --kirby-x-image-banner-min-height
+ */
+export const customCSS: Story = {
+  parameters: {
+    cssprops: {
+      disable: false,
+      'kirby-x-image-banner-min-height': {
+        control: 'text',
+        default: 'auto',
+        description: 'You can override the min-height of the image banner',
+      },
+    },
+  },
+  args: {
+    title: 'This card only has a title',
+    imagePath: 'assets/images/leaves.jpg',
+  },
+
+  render: (args) => ({
+    props: args,
+    template: `<kirby-x-image-banner ${argsToTemplate(args)}></kirby-x-image-banner>`,
```

Comment:
I think we should do like this to make it clear to the reader what is happening when they see this example on the docs page, otherwise they have to go to the specific story to explore the css props, because on the docs page it looks like this, as the css property is not applied before going to the panel: 

<img width="462" alt="image" src="https://github.com/user-attachments/assets/8fca568b-fae0-46d0-bc78-b925ae40293b">

```suggestion
    template: `<kirby-x-image-banner ${argsToTemplate(args)} style="--kirby-x-image-banner-min-height: auto;"></kirby-x-image-banner>`,
```

Reply:
Consider also surrounding with a div with sensible min-width, so the example shows the collapsed/condensed card no matter the width of the example container.

Reply:
> I think we should do like this to make it clear to the reader what is happening when they see this example on the docs page, otherwise they have to go to the specific story to explore the css props, because on the docs page it looks like this, as the css property is not applied before going to the panel:
> 
> <img alt="image" width="462" src="https://private-user-images.githubusercontent.com/42470636/382298657-8fca568b-fae0-46d0-bc78-b925ae40293b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA0NzI2MjEsIm5iZiI6MTczMDQ3MjMyMSwicGF0aCI6Ii80MjQ3MDYzNi8zODIyOTg2NTctOGZjYTU2OGItZmFlMC00NmQwLWJjNzgtYjkyNWFlNDAyOTNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTAxVDE0NDUyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFmOGYzMjM3YWVjOWRiNWZiZmQwYTU2NWVhOTYxYzU0OWE2NjEyMmYwY2JjNTViYTM0MWEzMzAzMzhhODVjYTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.wDi0oeRXlKtdrzB4hHi8euMaoLhTVx-gCHO7-BR5k5w">

Do you want me to remove the storybook plugin cssprops again? We won't be able to use the custom css property with cssprops once it's scored as an inline style.

Reply:
Played around with it to try and find a solution but noticed that the plugin does not work at all for me when deployed: https://665d7d16d6f1242580f1e9bf-rmvzyrqjzj.chromatic.com

Cant set the string at all... so probably better to remove it? Although I really like the intention you had.